docs/0-quickstart.ipynb

```diff
@@ -0,0 +1,489 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create and fit model\n",
+    "The `Blend` class represent the sources as a tree and has the machinery to fit all of the sources to the given images. In this example the code is set to run for a maximum of 200 iterations, but will end early if the likelihood and all of the constraints converge."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "blend = scarlet.Blend(sources, observation)\n",
+    "%time blend.fit(200)\n",
+    "print(\"scarlet ran for {0} iterations to logL = {1}\".format(len(blend.loss), -blend.loss[-1]))\n",
+    "plt.plot(-np.array(blend.loss))\n",
+    "plt.xlabel('Iteration')\n",
+    "plt.ylabel('log-Likelihood')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Interact with Results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### View the full model\n",
+    "\n",
+    "We could use `scarlet.display.show_scene` to render to entire scene, but it's instructive to see how the model and the comparison to observations is performed.\n",
+    "First we load the model for the entire scene, render it in the observation frame, and compute its residuals. We then show model and date with the same $sinh^{-1}$ stretch and the residuals with a linear stretch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the model and calculate the residual\n",
+    "model = blend.get_model()\n",
+    "model_ = observation.render(model)\n",
+    "residual = images-model_\n",
+    "\n",
+    "# Create RGB images\n",
+    "model_rgb = scarlet.display.img_to_rgb(model_, norm=norm)\n",
+    "residual_rgb = scarlet.display.img_to_rgb(residual)\n",
+    "\n",
+    "# Show the data, model, and residual\n",
+    "fig = plt.figure(figsize=(15,5))\n",
+    "ax = [fig.add_subplot(1,3,n+1) for n in range(3)]\n",
+    "ax[0].imshow(img_rgb)\n",
+    "ax[0].set_title(\"Data\")\n",
+    "ax[1].imshow(model_rgb)\n",
+    "ax[1].set_title(\"Model\")\n",
+    "ax[2].imshow(residual_rgb)\n",
+    "ax[2].set_title(\"Residual\")\n",
+    "\n",
+    "for k,component in enumerate(blend):\n",
+    "    y,x = component.center\n",
+    "    ax[0].text(x, y, k, color=\"w\")\n",
+    "    ax[1].text(x, y, k, color=\"w\")\n",
+    "    ax[2].text(x, y, k, color=\"w\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### View the source models\n",
+    "\n",
+    "We will now inspect the model for each source, in its original frame and in its observed frame by leveraging the `show_sources` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scarlet.display.show_sources(sources, \n",
```

Comment:
I'm not sure I agree with you, Fred. This method is for showing the sources, which are in boxes. It's an option to show them also in the rendered/observed frame. Introducing a new box for that is at least somewhat misleading (because it's either not the source box or it has to use a larger image for the rendered one so that the spatial scale is different). This is even more problematic for multi-resolution, where the spatial scale *is* different. Suggestions welcome.

Reply:
I guess what I was thinking is that all of the frames would show the larger box, which is the size of the box + the size of the PSF. This doesn't necessarily have to be part of this PR, but I'd like to see the _option_ of using a box that will fit the convolved model for a source.

Reply:
This requires us to know what the size of the observed PSF is. We don't. Alternatively, we could *determine* the box from the convolved model by thresholding at some low level. Again, none of that is essential.

Reply:
`show_sources` takes `observation` as an optional input, which includes the PSF image in each band. So I would argue that if `show_sources` receives an ` observation` and we are displaying the sources in the `observation` seeing, we should inject the source into a box that displays the entire source.